src/dg/dg.cpp

```diff
@@ -99,7 +99,7 @@ DGBase<dim,real,MeshType>::DGBase(
     , oneD_quadrature_collection(std::get<7>(collection_tuple))
     , oneD_face_quadrature(max_degree)
     , dof_handler(*triangulation, true)
-    , high_order_grid(std::make_shared<HighOrderGrid<dim,real,MeshType>>(grid_degree_input, triangulation, all_parameters->output_high_order_grid))
+    , high_order_grid(std::make_shared<HighOrderGrid<dim,real,MeshType>>(grid_degree_input, triangulation, all_parameters->check_valid_metric_Jacobian, all_parameters->renumber_dof_handler_Cuthill_Mckee, all_parameters->output_high_order_grid))
```

Comment:
use `do_renumber_dofs` and add a function argument `dof_renumbering_type` which is an enum; this would replace the current argument `renumber_dof_handler_Cuthill_Mckee` and allow for easy extension if we ever want to use different renumberings; can open an issue for this or resolve now

Reply:
https://github.com/dougshidong/PHiLiP/issues/220